### PR TITLE
Remove CaaSPv3

### PR DESCRIPTION
**Pull request by jsuchome**
Originally https://github.com/SUSE-Cloud/socok8s/pull/673

### CODE REVIEW

File: Jenkinsfile.integration [hunk 1]
```diff
@@ -23,7 +23,6 @@ pipeline {
     parameters {
```
Comment (toabctl) [inline Jenkinsfile.integration]: - Why has the commit msg (for 42cdde8ecfd6f56cacfd65dd96261fdfe44a17c4) a cherry-pick line? Doesn't make sense imo

File: run.sh [hunk 2]
```diff
@@ -69,13 +69,8 @@ case "$deployment_action" in
         deploy_ses
         ;;
     "deploy_caasp")
-        # this is CaaSP 3
         deploy_caasp
         ;;
-    "deploy_caasp4")
```
Comment (toabctl) [inline run.sh]: I would prefer to keep `caasp4` everywhere and just call it in `deploy_caasp4` , too. That way the PR will be pretty simple.

File: run.sh [hunk 3]
```diff
@@ -127,7 +127,7 @@ case "$deployment_action" in
         deploy_airship
         ;;
     "update_openstack")
-        deploy_airship update_airship_osh_site
+        deploy_airship update_airship_osh_site
```
Comment (toabctl) [inline run.sh]: unrelated change

File: Jenkinsfile.integration (hunk 1 above)
Comment (jsuchome) [reply Jenkinsfile.integration]: Because it was cherry-picked from the older one? I'm not sure I get your point

File: run.sh (hunk 2 above)
Comment (jsuchome) [reply run.sh]: Well it would be less simple now to revert previous changes :-)

Is there a better reason to keep caasp4 everywhere?

File: Jenkinsfile.integration (hunk 1 above)
Comment (jsuchome) [reply Jenkinsfile.integration]: removed
